lib/src/models/subreddit_moderation.dart

```diff
@@ -314,6 +328,72 @@ ModeratorActionType stringToModeratorActionType(String s) {
   }
 }
 
+SpamSensitivity stringToSpamSensitivity(String s) {
+  switch (s) {
+    case 'all':
+      return SpamSensitivity.all;
+    case 'low':
+      return SpamSensitivity.low;
+    case 'high':
+      return SpamSensitivity.high;
+    default:
+      throw DRAWInternalError('Invalid spam sensitivity type: $s.');
+  }
+}
+
+WikiAccessMode stringToWikiAccessMode(String s) {
+  switch (s) {
+    case 'anyone':
+      return WikiAccessMode.anyone;
+    case 'modonly':
+      return WikiAccessMode.modonly;
+    case 'disabled':
+      return WikiAccessMode.disabled;
+    default:
+      throw DRAWInternalError('Invalid Wiki Access Mode: $s.');
+  }
+}
+
+PostContentType stringToContentType(String s) {
+  switch (s) {
+    case 'any':
+      return PostContentType.any;
+    case 'self':
+      return PostContentType.self;
+    case 'link':
+      return PostContentType.link;
+    default:
+      throw DRAWInternalError('Invalid Post Content type: $s.');
+  }
+}
+
+SuggestedCommentSort? stringToSuggestedCommentSort(String? s) {
+  if (s == null) {
```

Comment:
I could add another option called 'None' but i think this is better

Note: this is null by default.